tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -1043,3 +1043,15 @@ def test_create_new_edit_perms_with_teams(self):
         assert response.status_code == 201, response.content
         assert response.data["permissions"]["isEditableByEveryone"] is False
         assert response.data["permissions"]["teamsWithEditAccess"] == [team1.id, team2.id]
+
+    def test_gets_dashboard_permissions_with_dashboard_list(self):
```

Comment:
Can you also add a test where you create the permissions, link it to the dashboard, and validate that the output when we _do_ have data is good? There should be examples of this in Harshi's previous tests in `test_organization_dashboard_details.py`